README.md

```diff
@@ -97,18 +97,30 @@ Options:
 -h              This help info
 ```
 
-Note that CFSSL is required if you want to support revocation. As noted above, do not use
-the TPM emulator option `-s` in production systems.
+Should you not have the Keylime repository on your local machine, you can
+use the `-k` flag which will download the software. In this case all you need
+is the `installer.sh` script locally.
+
+Note that CFSSL is required if you want to support revocation. As noted above, do not use the TPM emulator option `-s` in production systems.
+
+#### Installer Distribution coverage
+
+| Distribution  | Versions      | TPM2-Software   |
+| ------------- |:-------------:| -----:          |
+| CentOS        | 7 / 8         | Compiled        |
+| RHEL          | 7 / 8         | Compiled        |
+| Fedora        | 29 / 30 / 31  | Package Install |
+| Ubuntu        | 18.04         | Compiled        |
```

Comment:
For Ubuntu, should this be `18 LTS / 19` instead?

Reply:
Happy to put whatever findings you guys had during testing. 

Should I have put the minor versioning in place too (eg. 19.*04*)

Reply:
It's probably fine just to go with `18 LTS / 19`, but I'm not sure how significant the changes between 19.04 and 19.10 will be.